xds/server.go

```diff
@@ -380,17 +381,21 @@ func routeAndProcess(ctx context.Context) error {
 // xdsUnaryInterceptor is the unary interceptor added to the gRPC server to
 // perform any xDS specific functionality on unary RPCs.
 func xdsUnaryInterceptor(ctx context.Context, req interface{}, _ *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (resp interface{}, err error) {
-	if err := routeAndProcess(ctx); err != nil {
-		return nil, err
+	if env.RBACSupport {
```

Comment:
This also seems too broad.  Yes RBAC is the only interceptor we can install today, but IMO `routeAndProcess` should still run, and just not have any interceptors to call, since RBAC is disabled.

Reply:
Discussed this with Eric and Mark, the problem with Route Configuration is the logical clause that incoming RPC's have to match to a VH (based on :authority) and then a Route (based on properties about the incoming RPC). If the route matched to doesn't have a type of NonForwardingAction, the RPC gets rejected as per the addition to A36. Thus, this would be an extra step for the user to set up their service mesh and have RPC's proceed as they previously did for a logical no-op (the only thing RouteConfiguration does is determine HTTP Filter overrides).